src/qt/transactiondesc.cpp

```diff
@@ -48,7 +48,6 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTx& wtx, const i
     }
 }
 
-#ifndef ENABLE_BIP70
 // Takes an encoded PaymentRequest as a string and tries to find the Common Name of the X.509 certificate
 // used to sign the PaymentRequest.
 bool GetPaymentRequestMerchant(const std::string& pr, QString& merchant)
```

Comment:
Could this method not rather be removed as well, along with the one place where it is used (line 295, same file)? I guess you try to keep the PR small but I would probably still remove this.

Reply:
I think this is still needed to display the merchant name for existing txs made with bip 70?

Reply:
It's not the intent to remove this any time soon. Maybe in the far future on a wallet format change. This was added as *fallback* for removing of BIP70 in #16852, so that displaying of old transactions doesn't suffer.

Reply:
Ok, thanks for clarifying!